### PR TITLE
Adds pt_BR translation for some sections of the FAQ.

### PR DESCRIPTION
**Pull request by rascaraficci**
The following sections were translated to portuguese: General,
Usage and Devices.

### CODE REVIEW

File: source/locale/pt_BR/LC_MESSAGES/faq.po [hunk 1]
```diff
@@ -19,214 +19,281 @@ msgstr ""
...
+"A base da dojot é o Fiware, projeto também de código aberto, comprometido"
+" com a construção de um ecossistema aberto e sustentável em torno de "
+"padrões abertos, destinados a facilitar a criação de aplicações em "
+"diversos segmentos."
 
-#: ../../source/faq/faq.rst:41
+#: ../../source/faq/faq.rst:43
 msgid "Where can I get it?"
-msgstr ""
+msgstr "Onde eu consigo isso?"
```
Comment (giovannicuriel) [inline source/locale/pt_BR/LC_MESSAGES/faq.po]: Talvez "Onde eu posso baixar?"

File: source/locale/pt_BR/LC_MESSAGES/faq.po [hunk 2]
```diff
@@ -19,214 +19,281 @@ msgstr ""
...
+#: ../../source/faq/faq.rst:129
 msgid ""
 "Consider, for instance, an actual device with temperature and humidity "
 "sensors; it can be represented into dojot as a device with two attributes"
 " (one for each sensor). We call this kind of device as *regular device* "
 "or by its communication protocol, for instance, *MQTT device* or *CoAP "
 "device*."
 msgstr ""
+"Considere, por exemplo, um dispositivo real com sensores de temperatura e"
+" humidade; ele pode ser representado na dojot como um dispositivo com "
```
Comment (giovannicuriel) [inline source/locale/pt_BR/LC_MESSAGES/faq.po]: umidade

File: source/locale/pt_BR/LC_MESSAGES/faq.po [hunk 3]
```diff
@@ -250,277 +323,341 @@ msgid ""
...
+#: ../../source/faq/faq.rst:168
 msgid ""
 "Let’s take as example a weather station which measures temperature and "
 "humidity, and publishes them periodically through MQTT. First, you create"
 " a device of type MQTT with two attributes (temperature and humidity). "
 "Then you set your actual device to push the data to dojot. Here, you need"
 " to follow some rules:"
 msgstr ""
+"Tomemos como exemplo uma estação meteorólogica que monitora temperatura e"
+" humidade e publica essas medidas via MQTT periodicamente. Inicialmente, "
```
Comment (giovannicuriel) [inline source/locale/pt_BR/LC_MESSAGES/faq.po]: umidade

File: source/locale/pt_BR/LC_MESSAGES/faq.po [hunk 4]
```diff
@@ -250,277 +323,341 @@ msgid ""
...
+#: ../../source/faq/faq.rst:203
 msgid ""
 "If your device is able to send messages using MQTT (with JSON payload), "
 "CoAP or HTTP, there is a good chance that your device can be integrated "
 "with minor or no modifications whatsoever. The requirements for such "
 "integration is described in the question `How can I send MQTT data to "
 "dojot so that it appears on the dashboard?`_."
 msgstr ""
+"Se o seu dispositivo envia mensagens\t via MQTT (com `payload` do tipo "
+"JSON), CoAP ou HTTP, existe uma grande chance de ser possível intergra "
```
Comment (giovannicuriel) [inline source/locale/pt_BR/LC_MESSAGES/faq.po]: integrá-lo?

File: source/locale/pt_BR/LC_MESSAGES/faq.po [hunk 5]
```diff
@@ -250,277 +323,341 @@ msgid ""
...
+"descritos na questão `Como posso enviar dados via MQTT para a dojot de "
+"forma que apareçam no dashboard`_?."
 
-#: ../../source/faq/faq.rst:207
+#: ../../source/faq/faq.rst:209
 msgid ""
 "Is there any restrictions about the message my device will send to dojot?"
 " Format, size, frequency?"
 msgstr ""
+"Existem restrições para as menssagens enviadas pelo meu dispositivo para "
```
Comment (giovannicuriel) [inline source/locale/pt_BR/LC_MESSAGES/faq.po]: mensagens

File: source/locale/pt_BR/LC_MESSAGES/faq.po (hunk 1 above)
Comment (rascaraficci) [reply source/locale/pt_BR/LC_MESSAGES/faq.po]: Your suggestion is better.

File: source/locale/pt_BR/LC_MESSAGES/faq.po (hunk 2 above)
Comment (rascaraficci) [reply source/locale/pt_BR/LC_MESSAGES/faq.po]: You're right!

File: source/locale/pt_BR/LC_MESSAGES/faq.po (hunk 3 above)
Comment (rascaraficci) [reply source/locale/pt_BR/LC_MESSAGES/faq.po]: You're right!

File: source/locale/pt_BR/LC_MESSAGES/faq.po (hunk 5 above)
Comment (rascaraficci) [reply source/locale/pt_BR/LC_MESSAGES/faq.po]: You're right!

File: source/locale/pt_BR/LC_MESSAGES/faq.po (hunk 4 above)
Comment (rascaraficci) [reply source/locale/pt_BR/LC_MESSAGES/faq.po]: Ok

**Pull request merged**